ci-operator/step-registry/workers-rhel7/scaleup/workers-rhel7-scaleup-commands.sh

```diff
@@ -16,18 +16,46 @@ if ! whoami &> /dev/null; then
     fi
 fi
 
+# Install an updated version of the client
+mkdir -p /tmp/client
+curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz | tar --directory=/tmp/client -xzf -
```

Comment:
I dislike curling down clients.  Can this move to a separate step, "wait for RHEL scaleup" or whatever, that runs from the `cli` image or something else that has a copy of `oc` baked in?

Reply:
I don't like it either.  When implementing the workers-rhel7 steps I found there were more problems to be addressed.  Would you be fine with tabling the oc client issue until we can work on [this](https://issues.redhat.com/browse/CORS-1532)? 

I had to do the same [here](https://github.com/openshift/release/pull/11132) because we needed at 4.5.  Once the ansible image is updated we could switch the ssh-bastion step to use it instead of the upi-installer image.